dependencies.gradle

```diff
@@ -0,0 +1,23 @@
+ext {
+    versions = [
+            androidSdk    : [
+                    compile   : 27,
+                    min       : 23,
+                    target    : 27,
+                    buildTools: '26.0.2'
+            ],
+            androidSupport: '27.0.0',
+            kotlin        : '1.1.51'
+    ]
+
+    libraries = [
+            build             : [
+                    androidGradle: 'com.android.tools.build:gradle:3.0.1',
```

Comment:
this is where the ` apply plugin: 'com.android.application'` comes from. 3.0.1 is latest version I think 🙌